### ISSUE TITLE
Error message when filtering by onboarding status or adding onboarding columns

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#5104]
On the ChA admin page (/admin/chapter_ambassadors#/) on preview, I receive an error message when I complete the following tasks:

- Filter by onboarded status
- Filter by assigned to chapter status
- Add columns for remaining chapter ambassador onboarding tasks, remaining chapter onboarding tasks, or onboarded columns

These features are working on QA.

The following error in the airbrake channel is likely related:

ActionView::Template::Error: PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "chapter" LINE[Truncated]

**Comment by dboyer** [comment]
Tested on preview and confirmed I could now:

- Add the remaining chapter ambassador onboarding tasks, remaining chapter onboarding tasks, or onboarded columns
- Filter by not fully onboarded

@shaun-technovation I think there is still something off when I select the "yes, fully onboarded" option from the onboarded filter.

1. I load the page (/admin/chapter_ambassadors#/) and it says there are 15 ChAs
2. I select 'Yes, fully onboarded" from the onboarded filter
3. The results still say 15 ChAs
4. I switch the filter to "No, still onboarding" and it says 8 ChAs
5. I then went to the chapter admin page (/admin/chapters) where it says there are 15 chapters
6. I tried the onboarded filter and when I select "Yes, fully onboarded" it says 15 chapters but when I select "No, still onboarding" it also says 15 chapters

I'm guessing something is up with how it filters for whether the chapter is onboarded.
